Technical-Steering-Committee/branding.md

```diff
@@ -0,0 +1,9 @@
+# Branding 
+
+This policy builds upon and extends the [Linux Foundation trademark policy](https://www.linuxfoundation.org/legal/trademark-usage) and [Open Mainframe Project's Branding guidelines](https://www.openmainframeproject.org/branding-guidelines).
+
+All the projects by default use the Zowe logo and imagery approved by TSC. The projects don't create derivatives
```

Comment:
The wording is strange to me for a policy document. "all projects by default use" implies a fact that is desired but may not be true. To make that desire a policy, how about:

"All projects must use the [standard Zowe logo](https://artwork.openmainframeproject.org/projects/zowe/) ([github source](https://github.com/openmainframeproject/artwork/tree/main/projects/zowe)) and imagery approved by the Technical Steering Committee (TSC), Open Mainframe Project, and Linux Foundation. Projects must not create derivatives or custom logos except when granted through an exception by the TSC. Such exception will be granted in cooperation with the Linux Foundation [art team](https://github.com/openmainframeproject/artwork).

Reply:
Accepted your version.